squad_client/commands/version.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from squad_client.core.api import SquadApi
+from squad_client.core.command import SquadClientCommand
+from squad_client.version import __version__
+
+
+logger = logging.getLogger()
```

Comment:
Please use `__name__` for the logger name.

ex: `logger = logging.getLogger(__name__)`